lib/child-writer/writer-proxy.js

```diff
@@ -163,16 +163,45 @@ permissions.isElevated().then((elevated) => {
         error: error.message,
         data: error.stack
       })
-      child && child.kill()
+      child && child.kill('SIGINT')
       reject(error)
     }
 
+    /**
+     * @summary Create a signal handler that will
+     * either kill the child with the received signal,
+     * or exit, if the child is already killed
+     * @private
+     *
+     * @param {String} signal - signal
+     * @returns {Function}
+     *
+     * @example
+     * process.on('SIGINT', signalHandler('SIGINT'))
+     */
+    const signalHandler = (signal) => {
+      return () => {
+        if (child) {
+          child.kill(signal)
+        } else {
```

Comment:
Not sure if I'm getting this right, but shouldn't we call `process.exit()` also if the child exists? Basically, it looks like this shouldn't be an `else`, but something that happens even if there was a child.

Reply:
If the child exists, the child's exit event will cause this process to exit, after emitting the appropriate things